### DISCUSSION TITLE
AWS Batch + AWS Fargate & Copilot

### DISCUSSION THREAD

**Discussion by atelierJVA** [aws/copilot-cli#2797] [Ideas]
Hi,

Are you planning to integrate the creation of AWS Batch + AWS Fargate projects with Copilot?

Thanks

**Comment by efekarakus** [comment]
Hi @atelierJVA !

We're really interested in integrating Batch+Copilot 🌈 what are some usecases that you think would be cool to support?  

Today, while we don't have any patterns for AWS Batch, but you can create [scheduled jobs](https://aws.github.io/copilot-cli/docs/manifest/scheduled-job/) with Copilot that will result in: `CloudWatch Event -> Step Function State Machine -> Fargate Task`

**Reply by atelierJVA** [reply]
Hi @efekarakus ,

Thanks for the info about scheduled jobs with Copilot, looks really interesting.

Currently, my use cases with AWS Batch + AWS Fargate are the basic ones, as described in https://aws.amazon.com/blogs/aws/new-fully-serverless-batch-computing-with-aws-batch-support-for-aws-fargate/
Basically been able to define fargate compute environments (reserved & spot), job queues and job definitions, and then submit jobs to the job queues.

**Reply by efekarakus** [reply]
Thanks that make sense to me!
